tests/System.Buffers.Experimental.Tests/BufferExtensionTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.IO.Pipelines;
+using System.IO.Pipelines.Testing;
+using Xunit;
+
+namespace System.Buffers.Experimental.Tests
+{
+    public class BufferExtensionTests
+    {
+        [Fact]
+        public void ToSpanOnReadableBufferSingleSpan()
+        {
+            byte[] data = { 0, 1, 2, 3, 4, 5, 6, 7 };
+
+            var readable = BufferUtilities.CreateBuffer(data);
+            var span = readable.ToSpan();
```

Comment:
It does not run that version of ToSpan. It runs the one in buffer experimental. That ToSpan is internal and private to pipelines.